assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -52,10 +52,11 @@ function DashboardClicksWidget() {
 		const isDomainProperty = select( STORE_NAME ).isDomainProperty();
 		const referenceSiteURL = untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() );
 
+		const dateRange = select( CORE_USER ).getDateRangeDates( { compare: true } );
```

Comment:
Another thing missing from the IB (🤦‍♂️ ) is that we'll need to pass the `offsetDays` argument to match the behavior of how the date range is parsed in PHP (where the offset is 1). The default in the JS selector is `0`, so we'll need to pass that as `1`.